hospitalresponse.cpp

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <string.h>
+
+//input the name of the local emergency team
+//output total response time in seconds
+
+int main()
+{
+#define MAX_TEAMNAME_SIZE 100

```

Comment:
`#define` is C

`constexpr int MAX_TEAMNAME_SIZE{100}` is C++. TBH the define works in C++ as well.